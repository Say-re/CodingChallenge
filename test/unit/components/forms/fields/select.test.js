import React from 'react';
import { shallow } from 'enzyme';
import SelectOptionComponent from '../../../../../src/components/forms/fields/select';


describe('Select Option Component Tests', () => {
  test('SelectOptionComponent Snapshot', async () => {
    const wrapper = shallow(<SelectOptionComponent />)
    await expect(wrapper).toMatchSnapshot();
  })
})