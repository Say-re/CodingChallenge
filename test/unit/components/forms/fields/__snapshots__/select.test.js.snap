// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select Option Component Tests SelectOptionComponent Snapshot 1`] = `
<div
  id="selectOptions"
>
  <label
    for="job-select"
  >
    Select Job Title
  </label>
  <select
    id="job-select"
    name="jobs"
    onChange={[Function]}
    required={true}
  >
    <option
      key="0"
      value="Manager"
    >
      Manager
    </option>
    <option
      key="1"
      value="Engineer - Lead"
    >
      Engineer - Lead
    </option>
    <option
      key="2"
      value="Engineer - Mid-level"
    >
      Engineer - Mid-level
    </option>
    <option
      key="3"
      value="Engineer - Junior"
    >
      Engineer - Junior
    </option>
    <option
      key="4"
      value="Engineer - Front-end Focused"
    >
      Engineer - Front-end Focused
    </option>
    <option
      key="5"
      value="Engineer - Backend Focused"
    >
      Engineer - Backend Focused
    </option>
    <option
      key="6"
      value="Engineer - Fullstack"
    >
      Engineer - Fullstack
    </option>
  </select>
</div>
`;
